cassiopeia/solver/DistanceSolver.py

```diff
@@ -11,11 +11,17 @@
 import networkx as nx
 import numpy as np
 import pandas as pd
+import configparser
+import subprocess
+import os
+import tempfile
+import ete3
 
```

Comment:
nit: these imports should follow our Cassiopeia style - namely, `os` and `sys` should be imported first; then system-level imports; then packages installed manually. All should be alphabetical.

For example,

```
import os

import subprocess
import tempfile

import configparser
import ete3
import networkx as nx
import numpy as np
import pandas as pd
```